### PR TITLE
Recommend buildozer commands to resolve indirect or unused deps

### PR DESCRIPTION
**Pull request by pauldraper**

### CODE REVIEW

**Comment by andyscott** [comment]
Can you think of a good way to test for these output to ensure we don't break things down the road?

**Comment by pauldraper** [comment]
I think shell scripts are simple, everyone understand them immediately. And simple.

> Additionally I was thinking about rewriting the scripted runner as a full blown Scala program so it'll be easier to extend.

That seems like a lot of work, for implementation and for consumption, for `cat`, `rm`, `grep`, `sha1sum`.

The only real reason I could see is we wanted to run Windows, but Bazel already requires MSYS/Cygwin.

> I find shell scripts become a slippery slope to a having lots of messy tests down the road.

I agree the tests should be kept as simple as possible. Putting the complexity into a Scala runner though....not sure that helps much.

**Comment by pauldraper** [comment]
This is now ready (except for the open question of tests).

**Comment by pauldraper** [comment]
@andyscott I added a commit to use toolchains for the deps checker. IDK if they're really that useful; I would have just put selects into everywhere.

File: README.md [hunk 1]
```diff
@@ -49,8 +49,9 @@ http_archive(
   url = "https://github.com/andyscott/rules_scala_annex/archive/<commit>.zip",
 )
 
-load("@rules_scala_annex//rules/scala:workspace.bzl", "annex_scala_repositories")
+load("@rules_scala_annex//rules/scala:workspace.bzl", "annex_scala_repositories", "annex_scala_register_toolschains")
```
Comment (andyscott) [inline README.md]: Looks like you've got a typo

**Pull request merged**